packages/web/src/components/notification/Notifications/FavoriteNotification.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+
+import { Favorite } from 'common/store/notifications/types'
+
+import { EntityLink, useGoToEntity } from './EntityLink'
+import { NotificationBody } from './NotificationBody'
+import { NotificationFooter } from './NotificationFooter'
+import { NotificationHeader } from './NotificationHeader'
+import { NotificationTile } from './NotificationTile'
+import { TwitterShareButton } from './TwitterShareButton'
+import { UserNameLink } from './UserNameLink'
+import { UserProfileList } from './UserProfileList'
+import { IconFavorite } from './icons'
+import { formatOthersCount } from './utils'
+
+const messages = {
+  others: formatOthersCount,
+  reposted: ' reposted your '
+}
+
+type FavoriteNotificationProps = {
+  notification: Favorite
+}
+export const FavoriteNotification = (props: FavoriteNotificationProps) => {
+  const { notification } = props
+  const { users, entity, entityType, timeLabel, isRead } = notification
+  const [firstUser, ...otherUsers] = users
+  const otherUsersCount = otherUsers.length
+
+  const handleClick = useGoToEntity(entity, entityType)
+
+  return (
+    <NotificationTile notification={notification} onClick={handleClick}>
+      <NotificationHeader icon={<IconFavorite />}>
+        <UserProfileList users={users} />
+      </NotificationHeader>
+      <NotificationBody>
+        <UserNameLink user={firstUser} notification={notification} />
+        {otherUsersCount > 0 ? messages.others(otherUsersCount) : null}
+        {messages.reposted}
+        {entityType.toLowerCase()}
```

Comment:
nit: These all seem like part of one "message", can we have some method or something return a string with all of these templated together for clarity?

Reply:
yeah for sure! i was trying something out by putting it all in as react components, you think it's confusing ultimately? i guess it's hard to know that that is literally all just text huh? where it get's a lil tricky is when there is text that is also a link + tooltip, then i couldn't really template that in. thoughts?

Reply:
Yeah that's true... Generally I think having it all be one "message" makes it easier in the future for other translations to make it grammatically correct, otherwise the grammar is already baked in by the ordering of the rendered strings.

Just looked up https://react.i18next.com/latest/trans-component and maybe it's a non issue? Feel free to ignore for now and we can revisit when doing i18n